emva1288/process/plotting.py

```diff
@@ -477,7 +477,7 @@ class PlotVerticalSpectrogramDSNU(Emva1288Plot):
 
     name = 'Vertical spectrogram DSNU'
     xlabel = 'Cycles [periods/pixel]'
-    ylabel = 'Standard deviation and\nrelative presence of each cycle [DN]'
+    ylabel = 'Standard deviation and\nrelative presence of each cycle [e^{-}]'
```

Comment:
Once the units are consistent, change this so it formats correctly.
```suggestion
    ylabel = 'Standard deviation and\nrelative presence of each cycle [$e^-$]'
```